ntoskrnl/lpc/complete.c

```diff
@@ -469,6 +547,25 @@ NtCompleteConnectPort(IN HANDLE PortHandle)
         return STATUS_SUCCESS;
     }
 
+    /* Ensure client thread is on the reply chain */
+    if (Port->ConnectionPort)
+    {
+        PLIST_ENTRY Entry;
+        for (Entry = Port->ConnectionPort->LpcReplyChainHead.Flink; Entry != (PLIST_ENTRY)(&Port->ConnectionPort->LpcReplyChainHead.Flink); Entry = Entry->Flink)
```

Comment:
This line is so long, that you can basically place each part of the for-loop on its own line.

Reply:
Corrected.